piqe_ocp_lib/tests/resources/test_ocp_nodes.py

```diff
@@ -224,3 +224,81 @@ def test_execute_command_on_a_node(self, setup_params):
             decoded_str = command_api_response[2].decode("utf-8")
             logger.info("Decoded stderr string {}".format(decoded_str))
             assert decoded_str.find("executable file not found in $PATH") >= 1
+
+    def test_get_all_master_nodes(self, setup_params):
+        """
+        Verify that a list of all master nodes is returned
+        1. Call get_master_nodes method
+        2. Verify that the response object is of kind NodeList
+        :param setup_params:
+        :return:
+        """
+        node_api_obj = setup_params["node_api_obj"]
+        # Get all master nodes
+        api_response = node_api_obj.get_master_nodes()
+        node_count = len(api_response.items)
+        logger.info("{} master nodes returned in the list".format(node_count))
+        assert api_response.kind == "NodeList"
```

Comment:
I think it would be useful to have an additional check to assert that metadata.labels contains the master label